test/integration/password_reset_test.rb

```diff
@@ -97,6 +97,29 @@ def forgot_password_with(email)
     assert page.has_content?("Sign out")
   end
 
+  test "resetting password when webauthn is enabled" do
```

Comment:
Just an FYI, @simi is reorganizing integration/system tests https://github.com/rubygems/rubygems.org/pull/3374 so browser tests like this one will be moved to the `/system` test folder. Depending on which changes get merged first, this test might need to be moved under `test/system/sign_in_webauthn_test.rb`